examples/service-clients/odsp-client/shared-tree-demo/src/fluid.ts

```diff
@@ -0,0 +1,52 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { OdspClient, OdspContainerServices } from "@fluid-experimental/odsp-client";
+import { ContainerSchema, IFluidContainer } from "@fluidframework/fluid-static";
+import { SharedTreeFactory } from "@fluid-experimental/tree2";
+import { clientProps } from "./clientProps";
+
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+class SharedTree {
+	public static getFactory(): SharedTreeFactory {
+		return new SharedTreeFactory();
+	}
+}
```

Comment:
This doesn't feel like something we'd want to ultimately recommend - no fix needed in this PR but maybe log an item to review other options.